packages/shared/src/utils/benchmark-handler/benchmark-handler.ts

```diff
@@ -0,0 +1,38 @@
+type TComponetnPhase = 'mount' | 'update' | 'nested-update';
+
+type TBenchmarkHandler = (
+    id: string,
+    phase: TComponetnPhase,
+    actual_time: number,
+    base_duration: number,
+    start_time: number,
+    commit_time: number
+) => void;
+
+/**
+ * The function is intended to be used as a callback for the React Profiler.
```

Comment:
@amir-deriv please remove this js docs related code as you've already typed them using TS